packages/react-next/src/components/Autofill/Autofill.tsx

```diff
@@ -53,7 +65,7 @@ export class Autofill extends React.Component<IAutofillProps, IAutofillState> im
   }
 
   public get value(): string {
-    return this._value;
+    return this.state.inputValue || (this.props.value as string) || '';
```

Comment:
It looks like `InputHTMLAttributes` `value` prop can be `string | string[] | number`. If Autofill only supports `string`, then you should add it to `IAutofillProps` as `value?: string;` to restrict its type.

Reply:
I tried doing this and there is a surprising amount (type wise) that relies on this being `string | string[] | number`. I think for this release we should leave it as is to minimize the breaks that it could cause. @behowell do you have thoughts?

Reply:
That makes sense. But in that case, then this function should also handle the other types that value might be: `number` or `string[]`.

Reply:
I'm not sure how it could actually end up as any of those other types since the autofill input will always just be a text field. I suspect InputHTMLAttributes covers a broader range of components.

Reply:
We should probably explicitly add `value?: string` to `IAutofillProps` instead of using the default typing from `InputHTMLAttributes`.

Reply:
Yes, that was my initial suggestion :)

Reply:
Hmm seems I can't read today, sorry! So there are now two votes for an explicit typing. :)

Reply:
Yeah, I had tried that, there are a variety of things that rely on HTMLinputAttrbutes (or whatever that is) and that means that that set of props can't be passed right into the autofill. I think we can do that, but we would need to change it for a variety of thigns.

Reply:
@ecraig12345 Ben and I worked on a solution and we're going to keep that signature for now but handle the cases and warn for it.